### DISCUSSION TITLE
Enhanced Drafts

### DISCUSSION THREAD

**Discussion by paulux84** [outline/outline#6728] [Ideas]
Currently, outline platform allows users to save drafts of unpublished documents, but lacks the capability to save drafts for already published documents. This limitation becomes significant when users need time to modify (potentially collaboratively), review, and then publish updates to existing documents. The 'Enhanced Drafts' feature will enable users to create and save drafts of updates to published documents, fostering a more flexible and efficient workflow. This allows for thorough editing and collaboration before finalizing changes, ensuring that the published version is of the highest quality

**Comment by tommoor** [comment]
Definitely in the plans to have draft changes that can be merged into the main document, potentially once approved

**Reply by paulux84** [reply]
Hi, have you a relese date for this function?

**Reply by paulux84** [reply]
@tommoor hi, have you any news about this feature? 
Thanks
